pkg/api/dashboard.go

```diff
@@ -44,7 +44,7 @@ func dashboardGuardianResponse(err error) Response {
 }
 
 func GetDashboard(c *middleware.Context) Response {
-	dash, rsp := getDashboardHelper(c.OrgId, c.Params(":slug"), 0)
+	dash, rsp := getDashboardHelper(c.OrgId, c.Params(":slug"), 0, c.Params(":uid"))
```

Comment:
May want to create two different GetDashboard functions depending on if we want to get by slug or by uid so we dont have to handle both cases in the same func.